modules/ROOT/pages/migration-prep.adoc

```diff
@@ -46,24 +42,23 @@ Next, you need to make sure your deployment environment is ready. Depending on w
 |CloudHub is Mule 4 ready.
 
 |Hybrid Deployment
-|You must install the standalone version of Mule 4 your servers.
+|Install the standalone version of Mule 4 on your servers.
```

Comment:
Install Mule 4 on your servers. (when we talk about install I think it is implicit that it is standalone mule runtime right?)

Reply:
done via suggestion